mlrun/projects/project.py

```diff
@@ -2884,6 +2885,69 @@ def _init_function_from_obj(func, project, name=None):
     return name or func.metadata.name, func
 
 
+def _mask_credentials_in_function_object(function):
+    env_to_mask = [
+        "V3IO_ACCESS_KEY",
+        "V3IO_USERNAME",
+        "V3IO_PASSWORD",
+        "V3IO_API",
+        "V3IO_FRAMESD",
```

Comment:
I get it that this is instead of the `V3IO` prefix - wouldn't it be better to just check against the prefix + a short list of known secrets? It will make the code a bit less elegant, but this list is not elegant by itself (and error prone - what if we forgot something?)